src/main/java/org/sasanlabs/service/vulnerability/jwt/JWTVulnerability.java

```diff
@@ -662,4 +669,51 @@ private ResponseEntity<GenericVulnerabilityResponseBean<String>> getJWTResponseB
                         true, token, true, CollectionUtils.toMultiValueMap(headers));
         return responseEntity;
     }
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.HEADER_INJECTION,
+            description = "HEADER_INJECTION_VULNERABILITY_EXAMPLE")
```

Comment:
please update the Locale Key for description.

Reply:
Thanks, I have updated it